src/join_mesh.py

```diff
@@ -169,9 +171,13 @@ def join_mesh_recovery(prefix : str, partitions : int, recoveryPath : str):
 
         # Extract Global IDs
         array_data = part_point_data.GetArray("GlobalIDs")
+        # Check if GlobalIDs exist if not do partition-wise merge
+        if array_data is None:
+            logging.warning("GlobalIDs not found recovery merge is not possible.")
```

Comment:
```suggestion
            logging.warning("GlobalIDs were not found, a recovery merge is not possible.")
```
If you have one, could you give a reason for the missing `GlobalDs`?

Reply:
Can be anything, just right now we know that it is removed after mapping.